src/SystemView.cpp

```diff
@@ -50,19 +51,245 @@ static const int MAX_TRANSITION_FRAMES = 60;
 
 static const float ATLAS_SCROLL_SENS = .1f;
 
-void SystemView::InputBindings::RegisterBindings()
-{
-	mapViewPitch = AddAxis("BindMapViewPitch");
-	mapViewYaw = AddAxis("BindMapViewYaw");
-	mapViewZoom = AddAxis("BindMapViewZoom");
+namespace {
+	static bool too_near(const vector3f &a, const vector3f &b, const vector2f &gain, float width)
+	{
+		return std::abs(a.x - b.x) < gain.x && std::abs(a.y - b.y) < gain.y
+			// Gliese852:
```

Comment:
Just checking if we want to reference people in the source? 
(@Gliese852 has been pinged in source)

Reply:
This was just to indicate the original author of the comment/method given that it's more difficult to git blame the line since I did violence to the source file. The depth-based heuristic probably needs a bit more tweaking, and I'd prefer later contributors know where to start looking if they need more details.